docs/components/optimize/what-is-optimize.md

```diff
@@ -0,0 +1,26 @@
+---
+id: what-is-optimize
+title: What is Optimize?
+---
+
```

Comment:
Could you also add an admonition and link to the Optimize guide?

Reply:
Done and done!